forward.h

```diff
@@ -114,7 +114,7 @@ static inline int msg_send( struct socket_info* send_sock, int proto,
 	/* the raw processing callbacks are free to change whatever inside
 	 * the buffer further use out_buff.s and at the end try to free out_buff.s
 	 * if changed by callbacks */
-	if (proto != PROTO_BIN)
+	if ((proto != PROTO_BIN) && (proto != PROTO_HEP_TCP) && (proto != PROTO_HEP_UDP))
```

Comment:
@lemenkov , just to make things clear here, maybe we should add in `ip_addr.h` some new macro to tell if a protocol is SIP based or not - this is actually what we need here, to rule out the none-SIP protos.
Maybe something like `PROTO_IS_SIP()` ?

Reply:
@bogdan-iancu yes  this would be better!

Reply:
Could you check the eee990d commit (master) as an alternative to your patch? Upon your confirmation and I will do the backport. Thanks @lemenkov

Reply:
@bogdan-iancu looks good to me (rolled into production for a while now)